<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="W1751816" name="IDPMMDRest.IDPMMDRest.IDPMMDRest" tests="2" failures="0" timestamp="12 Nov 2019 06:35:39 GMT" time="7.973" errors="1">
  <testcase name="get1" time="2.187" classname="IDPMMDRest.IDPMMDRest.IDPMMDRest">
    <error type="org.json.JSONException" message="Missing &#039;;&#039; in XML entity: &amp;forceauthn at 1395 [character 1140 line 2]">
      <![CDATA[org.json.JSONException: Missing ';' in XML entity: &forceauthn at 1395 [character 1140 line 2]
	at org.json.JSONTokener.syntaxError(JSONTokener.java:433)
	at org.json.XMLTokener.nextEntity(XMLTokener.java:136)
	at org.json.XMLTokener.nextContent(XMLTokener.java:111)
	at org.json.XML.parse(XML.java:256)
	at org.json.XML.parse(XML.java:272)
	at org.json.XML.parse(XML.java:272)
	at org.json.XML.parse(XML.java:272)
	at org.json.XML.parse(XML.java:272)
	at org.json.XML.parse(XML.java:272)
	at org.json.XML.toJSONObject(XML.java:356)
	at IDPMMDRest.IDPMMDRest.IDPMMDRest.get1(IDPMMDRest.java:76)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- get1 -->
  <testcase name="get" time="5.786" classname="IDPMMDRest.IDPMMDRest.IDPMMDRest"/>
</testsuite> <!-- IDPMMDRest.IDPMMDRest.IDPMMDRest -->
